lib/_http_client.js

```diff
@@ -173,8 +173,10 @@ function ClientRequest(input, options, cb) {
 
   if (options.path) {
     const path = String(options.path);
-    if (RegExpPrototypeExec(INVALID_PATH_REGEX, path) !== null)
+    if (RegExpPrototypeExec(INVALID_PATH_REGEX, path) !== null) {
+      debug(`Path contains unescaped characters: "${path}"`);
```

Comment:
Please use positional arguments to avoid generating this string even when logging is disabled.